packages/dd-trace/src/telemetry/send-data.js

```diff
@@ -1,4 +1,20 @@
 const request = require('../exporters/common/request')
+
+function getHeaders (config, application, reqType) {
+  const headers = {
+    'content-type': 'application/json',
+    'dd-telemetry-api-version': 'v1',
+    'dd-telemetry-request-type': reqType,
+    'dd-client-library-language': application.language_name,
```

Comment:
Why are we repeating this data that is always available as other headers?

Reply:
`application` object, which contains language and version, is included in the request payload but apparently `dd-client-library-language` and `dd-client-library-version` headers are mandatory according to the spec. 

I gave you a reason in this [comment](https://github.com/DataDog/dd-trace-js/pull/3055#discussion_r1185954314)